### PR TITLE
support azure elb sync

### PR DESCRIPTION
**Pull request by tb365**
**What this PR does / why we need it**:
* support azure elb sync

**Does this PR need to be backport to the previous release branch?**:

/area region
/cc @zexi @ioito

### CODE REVIEW

File: pkg/multicloud/azure/loadbalancerlistener.go [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+func (self *SLoadBalancerListener) GetId() string {
+	return self.ID
+}
+
+func (self *SLoadBalancerListener) GetName() string {
+	return self.Name
+}
+
+func (self *SLoadBalancerListener) GetGlobalId() string {
+	return self.GetId()
```
Comment (ioito) [inline pkg/multicloud/azure/loadbalancerlistener.go:111]:
```suggestion
	return strings.ToLower(self.GetId())
```

File: pkg/multicloud/azure/loadbalancerlistenerrule.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+func (self *SLoadbalancerListenerRule) GetId() string {
+	return self.ID
+}
+
+func (self *SLoadbalancerListenerRule) GetName() string {
+	return self.Name
+}
+
+func (self *SLoadbalancerListenerRule) GetGlobalId() string {
+	return self.GetId()
```
Comment (ioito) [inline pkg/multicloud/azure/loadbalancerlistenerrule.go:40]:
```suggestion
	return strings.ToLower(self.GetId())
```

File: pkg/multicloud/azure/loadbalancer.go [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+func (self *SLoadbalancer) GetId() string {
+	return self.ID
+}
+
+func (self *SLoadbalancer) GetName() string {
+	return self.Name
+}
+
+func (self *SLoadbalancer) GetGlobalId() string {
+	return self.GetId()
```
Comment (ioito) [inline pkg/multicloud/azure/loadbalancer.go:41]:
```suggestion
	return strings.ToLower(self.GetId())
```

File: pkg/multicloud/azure/loadbalancerbackend.go [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+func (self *SLoadbalancerBackend) GetId() string {
+	return self.ID + "::" + strconv.Itoa(self.BackendPort)
+}
+
+func (self *SLoadbalancerBackend) GetName() string {
+	return self.Name
+}
+
+func (self *SLoadbalancerBackend) GetGlobalId() string {
+	return self.GetId()
```
Comment (ioito) [inline pkg/multicloud/azure/loadbalancerbackend.go:34]:
```suggestion
	return strings.ToLower(self.GetId())
```

File: pkg/multicloud/azure/loadbalancerbackendgroup.go [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+func (self *SLoadbalancerBackendGroup) GetName() string {
+	if self.HttpSettings != nil {
+		return self.Pool.Name + "::" + self.HttpSettings.Name
+	}
+
+	return self.Pool.Name + "::" + strconv.Itoa(self.DefaultPort)
+}
+
+func (self *SLoadbalancerBackendGroup) GetGlobalId() string {
+	return self.GetId()
```
Comment (ioito) [inline pkg/multicloud/azure/loadbalancerbackendgroup.go:45]:
```suggestion
	return strings.ToLower(self.GetId())
```

File: pkg/multicloud/azure/loadbalancercert.go [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+func (self *SLoadbalancerCert) GetId() string {
+	return self.ID
+}
+
+func (self *SLoadbalancerCert) GetName() string {
+	return self.Name
+}
+
+func (self *SLoadbalancerCert) GetGlobalId() string {
+	return self.GetId()
```
Comment (ioito) [inline pkg/multicloud/azure/loadbalancercert.go:37]:
```suggestion
	return strings.ToLower(self.GetId())
```

File: pkg/multicloud/azure/loadbalancerlistener.go (hunk 1 above)
Comment (tb365) [reply pkg/multicloud/azure/loadbalancerlistener.go]: fix

**Pull request merged**